tests/test_flake8_trio.py

```diff
@@ -366,7 +366,8 @@ def assert_correct_lines_and_codes(errors: Iterable[Error], expected: Iterable[E
                 sep=" | ",
                 file=sys.stderr,
             )
-    assert not any_error
+            error_count += abs(error_dict[line][code] - expected_dict[line][code])
+    assert error_count == 0
```

Comment:
added to be able to see if I was actually making progress when fixing a bug and the number of errors go from a double-digit number to another double-digit number. Also for the fun of it